src/client/src/containers/Licenses/index.tsx

```diff
@@ -14,33 +14,57 @@ import {
   IVisitedPages
 } from "../../selectors/wizardNavigationSelector";
 import { AppState } from "../../reducers";
+import * as ModalActions from "../../actions/modalActions/modalActions";
+import { ThunkDispatch } from "redux-thunk";
+import RootAction from "../../actions/ActionType";
+import { IPrivacyModalData } from "../PrivacyModal";
 
 interface IStateProps {
   frameworkLicenses: string[];
   pageLicenses: ILicenseObject[];
   isVisited: IVisitedPages;
 }
 
-type Props = IStateProps & InjectedIntlProps;
+interface IDispatchProps {
+  openPrivacyModal: (license: IPrivacyModalData | undefined) => any;
+}
+
+type Props = IStateProps & IDispatchProps & InjectedIntlProps;
 
 const messages = defineMessages({
   licenses: {
     id: "licenses.licenses",
     defaultMessage: "Licenses"
+  },
+  redirectLinkLabel: {
+    id: "licenses.redirectLinkLabel",
+    defaultMessage: "{licenseName} license link"
   }
 });
 
 const Licenses = ({
   frameworkLicenses,
   pageLicenses,
   isVisited,
-  intl
+  intl,
+  openPrivacyModal
 }: Props) => {
   const LinkRenderer = (props: any) => {
     return (
-      <a href={props.href} className={styles.link}>
+      <button
+        className={styles.licenseButton}
+        onClick={() =>
```

Comment:
NIT: quoted from tslint: 
Lambdas are forbidden in JSX attributes due to their rendering performance impacttslint(jsx-no-lambda)
A new instance of an anonymous method is passed as a JSX attribute: () => openP...tslint(react-this-binding-issue)